kouncil-frontend/apps/kouncil/src/app/schemas/form/form/schema-form.component.ts

```diff
@@ -152,7 +164,34 @@ export class SchemaFormComponent implements OnInit {
     });
   }
 
+  private defineDisabled() {
+    Object.keys(this.schemaForm.controls).forEach(key => {
+      if (
+        (ViewMode.VIEW === this.viewMode && key !== 'version')
```

Comment:
If the component would be splitted into three as I said earlier these conditions would be much simpler and easier to read :P